2021/29xxx/CVE-2021-29427.json

```diff
@@ -1,18 +1,88 @@
 {
```

Comment:
CNA(s) of user (GitHub_M) do not match owning CNA.

Reply:
👋 Hi I believe the GitHub CNA was assigned this CVE in service request 1051273 on 2021-03-30.  Can you check on the MITRE side and confirm whether this validation is accurate?